druid/examples/calc.rs

```diff
@@ -200,9 +200,9 @@ fn build_calc() -> impl Widget<CalcState> {
 }
 
 fn main() {
-    let window = WindowDesc::new(build_calc).title(
-        LocalizedString::new("calc-demo-window-title").with_placeholder("Simple Calculator"),
-    );
+    let window = WindowDesc::new(build_calc)
+        .title(LocalizedString::new("calc-demo-window-title").with_placeholder("Simple Calculator"))
+        .with_min_size((250.0, 350.0));
```

Comment:
can we move this to another example? I'd like to make calc be non-resizable (see #683)

Reply:
Sure, maybe the flex example?

Reply:
I like that idea!